src/js/vre/chapter31/components/EducationPeriodView.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function EducationView({ formData }) {
```

Comment:
Think this should be `EducationPeriodView`

Reply:
Good catch @rianfowler, changed.